Makefile

```diff
@@ -7,12 +7,17 @@ OCAML_LIB_DIR=$(shell ocamlc -where)
 
 CTYPES_LIB_DIR=$(shell ocamlfind query ctypes)
 
-OCAMLBUILD=CTYPES_LIB_DIR=$(CTYPES_LIB_DIR) OCAML_LIB_DIR=$(OCAML_LIB_DIR) \
-	ocamlbuild -use-ocamlfind -classic-display
+LWT_LIB_DIR=$(shell ocamlfind query lwt)
+
+OCAMLBUILD=CTYPES_LIB_DIR=$(CTYPES_LIB_DIR) \
+           LWT_LIB_DIR=$(LWT_LIB_DIR)       \
+           OCAML_LIB_DIR=$(OCAML_LIB_DIR)   \
+             ocamlbuild -use-ocamlfind -classic-display
 
 WITH_UNIX=$(shell ocamlfind query \
             ctypes unix unix-type-representations unix-errno.unix \
             > /dev/null 2>&1 ; echo $$?)
+WITH_LWT=$(shell ocamlfind query lwt unix > /dev/null 2>&1 ; echo $$?)
```

Comment:
The `lwt` depopt isn't too bad: the `unix-dirent.lwt` subpackage will be built iff `lwt` is installed.  For the `unix-dirent.unix` subpackage the relationship between depopts and build outputs is a bit less obvious.

One possibility is to have a virtual OPAM package `ocaml-unix-dirent-unix` or somesuch, which depends non-optionally on `ctypes`, `unix-errno` and `unix-type-representations`, and make *that* package the depopt of this package (`ocaml-unix-dirent`).